src/python/bot/fuzzers/syzkaller/fuzzer.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Google LLC
```

Comment:
nit: I know you used existing files as inspiration, but they were named for legacy reasons. Let's call this file "runner.py".

Reply:
Done.